docs/bigquery_to_vcf.md

```diff
@@ -122,7 +123,8 @@ python -m gcp_variant_transforms.bq_to_vcf \
   --output_file gs://BUCKET/loaded_file.vcf \
   --representative_header_file gs://BUCKET/representative_header_file.vcf \
   --project "${GOOGLE_CLOUD_PROJECT}" \
-  --temp_location gs://BUCKET/temp \
+  --region "${GOOGLE_CLOUD_REGION}" \
```

Comment:
Can't comment on the line above, but I think we still need temp_location for DirectRunner, no?